.ci/pipeline_definitions

```diff
@@ -49,3 +49,15 @@ etcd-druid:
               channel_name: 'k8s-etcd'
               slack_cfg_name: 'scp_workspace'
         component_descriptor: ~
+    create_update_pull_requests:
+      repo:
+        trigger: false
+        disable_ci_skip: true
+      steps: ~
+      traits:
+        component_descriptor: ~
+        update_component_deps:
+          merge_policy: 'auto_merge'
```

Comment:
Should it be `manual`?

Reply:
Either that or leave it out , e.g.: `update_component_deps: ~`